src/screens/TopicPageScreen/elements/Empty.js

```diff
@@ -4,6 +4,7 @@ import {
 } from 'react-native';
 import { Gap } from '../../../components';
 import { fonts } from '../../../utils/fonts';
+import { COLORS } from '../../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `COLORS` from the theme file has been added. This is a good practice as it centralizes the color definitions and makes the code more maintainable.
```diff
+ import { COLORS } from '../../../utils/theme';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->